### DISCUSSION TITLE
drag to create events in calendar?

### DISCUSSION THREAD

**Discussion by salekin02** [ant-design/ant-design#51473] [Q&A]
Is is possible to create events on multiple date in calendar? and how the event title in between dates?

https://ant.design/components/calendar#calendar-demo-select

example: https://fullcalendar.io/demos

**Comment by petercat-assistant** [comment]
Ant Design's Calendar component does not natively support creating events on multiple dates with drag-and-drop functionality like FullCalendar. The Calendar component in Ant Design is more focused on displaying dates and allowing for simple date selection.

To achieve functionality similar to FullCalendar, where you can drag to create events spanning multiple dates, you would need to implement custom logic. This might involve:

1. **Custom Event Handling**: Implementing event handlers to detect drag actions across dates.
2. **State Management**: Using state management to track the start and end dates of the drag action.
3. **Rendering Events**: Custom rendering logic to display events spanning multiple dates.

For a more feature-rich calendar with drag-and-drop capabilities, you might consider integrating a library like FullCalendar into your project alongside Ant Design for other UI components.

If you have further questions or need assistance with a specific implementation, feel free to ask!

For further assistance, please describe your question in the comments and @petercat-assistant to start a conversation with me.

**Comment by salekin02** [comment]
can i show selected dates with one single title? like fullcalendar does?

<img width="923" alt="Screenshot 2024-11-03 at 2 51 55 PM" src="https://github.com/user-attachments/assets/781e254d-b1d9-4118-92c7-d74966bbc6c9">
